CI/flatpak/com.obsproject.Studio.json

```diff
@@ -237,6 +237,32 @@
         }
       ]
     },
+    {
```

Comment:
Tiny commit message nits:
 * In the title, please use: `CI: Add PipeWire 0.3.40 to Flatpak`
 * `deamon` → `daemon`
 * `freedesktop flatpak sdk` → `Freedesktop Flatpak SDK`

Reply:
Done, but commit is retracted until #5131 .